pvlib/pvsystem.py

```diff
@@ -2845,6 +2845,80 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
     return pdc
 
 
+def _infer_k_huld(cell_type):
+    # from PVGIS documentation, Section 5.2.3, accessed 12/22/2023
+    huld_params = {'cSi': (-0.017237, -0.040465, -0.004702, 0.000149,
+                           0.000170, 0.000005),
+                   'CIS': (-0.005554, -0.038724, -0.003723, -0.000905,
+                           -0.001256, 0.000001),
+                   'CdTe': (-0.046689, -0.072844, -0.002262, 0.000276,
+                            -0.000159, -0.000006)}
+    return huld_params[cell_type]
+
+
+def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
+    r"""
+    Power (DC) using the Huld model.
+
+    The Huld model [1]_ is given by
+
+
+    .. math::
+
+        P_{dc} = G` ( P_{dc0} k_1 log(G') + k_2 log^2 (G') + k_3 T' log(G') +
+                 k_5 T' log^2 (G') + k_6 T'^2)
+
+        G' = \frac{G_{poa eff}}{1000}
+
+        T' = T_{mod} - 25 \degree C
```

Comment:
```suggestion
        T' = T_{mod} - 25 °C
```

I think mathjax doesn't have the degree symbol.  I think using unicode should be okay here?